src/data/script_i18n.ts

```diff
@@ -13,7 +13,7 @@ import {
   VariablesAction,
   EventAction,
 } from "./script";
-import { isDynamicTemplate } from "./template";
+import { isTemplate } from "./template";
```

Comment:
To be honest, having it in `common/string` made more sense as it is not fetching data. Just like we probably should move the i18n methods to `i18n/script.ts` etc ?